cmplxfoil/CMPLXFOIL.py

```diff
@@ -34,6 +34,32 @@ class is to provide an easy to use wrapper for xfoil for intergration
 import numpy as np
 from baseclasses import BaseSolver
 
+# Handle plotting imports
+plt = None
+try:
+    import matplotlib as mpl
+    plt = mpl.pyplot
+    mpl_lines = mpl.lines
+except ImportError:
+    pass
+
+# If the matplotlib import worked, try niceplots
+if plt:
+    try:
+        import niceplots as nice
+
+        nice.setRCParams()
+        plt.rcParams.update({"font.size": 18})
+        colors = nice.get_niceColors()
+        color = colors["Blue"]
+        cpUpColor = colors["Blue"]
+        cpLowColor = colors["Red"]
+    except ImportError:
+        print("Install niceplots for nice looking airfoil figures")
```

Comment:
Don't print the message here, because this will be shown every time people use this code even if they do not plan to plot anything. You can probably do a similar check for `nice` in the plotting routine and print messages there.

Reply:
Fixed

Reply:
This will actually print many many times if matplotlib is installed but niceplots isn't as it generates plots during the optimization

Reply:
Ah I see, maybe move it to the public facing function instead of the private function?

Reply:
The public facing function `plotAirfoil` is what's called every iteration. We could alternatively use logic to see if it's already been printed but I think that's too complicated for this very simple announcement. Can we just keep the one print statement at the beginning of the file?